src/main/java/cz/xtf/radanalytics/oshinko/deployment/template/python/OshinkoPythonSpark.java

```diff
@@ -0,0 +1,88 @@
+package cz.xtf.radanalytics.oshinko.deployment.template.python;
+
+import cz.xtf.radanalytics.oshinko.deployment.template.BaseTemplateDeployment;
+import cz.xtf.radanalytics.util.configuration.RadanalyticsConfiguration;
+
+import java.util.HashMap;
+import java.util.Map;
+
+public class OshinkoPythonSpark extends BaseTemplateDeployment {
+
+	private static final String RESOURCES_URL = RadanalyticsConfiguration.templateOshinkoPythonSpark();
+	private static final String TEMPLATE_FILE_NAME = "pythonbuilddc.json";
+
+	public static void deploy(Map<String, String> parameters) {
+		deploy(RESOURCES_URL, TEMPLATE_FILE_NAME, parameters);
+	}
+
+	public static void deployWinemap(){
```

Comment:
The same, I'd keep only general methods in here (eg. deploy())
Please move deployWinemap(), deployGrafzahl(), etc. to the test clases.